cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/batch/distributed/DistributedMapReduceTaskContextProvider.java

```diff
@@ -62,12 +65,15 @@ protected void startUp() throws Exception {
     try {
       List<ListenableFuture<State>> startFutures = Services.chainStart(zkClientService,
                                                                        kafkaClientService,
-                                                                       metricsCollectionService).get();
+                                                                       metricsCollectionService,
+                                                                       authorizationEnforcementService).get();
       // All services should be started
       for (ListenableFuture<State> future : startFutures) {
         Preconditions.checkState(future.get() == State.RUNNING,
-                                 "Failed to start services: zkClient %s, kafkaClient %s, metricsCollection %s",
-                                 zkClientService.state(), kafkaClientService.state(), metricsCollectionService.state());
+                                 "Failed to start services: zkClient %s, kafkaClient %s, metricsCollection %s, " +
+                                   "authorizationEnforcementService %s",
+                                 zkClientService.state(), kafkaClientService.state(), metricsCollectionService.state(),
```

Comment:
nit: unrelated to your change, but we probably don't need to do a ``.state()`` for any of these. Just print them as is, and Guava service will print it as ``<Service Name> [Service State]``. That way you can remove the literals ``zkClient``, ``kafkaClient``, etc?